TestServer/Unit/BookingServiceValidationTests.cs

```diff
@@ -89,4 +94,47 @@ public void ValidateBookingRequest_InvalidTimeAndSeatAlreadyBooked_ReturnsErrorM
         Assert.Contains("Booking date exceeds the latest allowed booking date.", result);
         Assert.Contains($"Seat {bookingRequest.SeatId} is already booked for the specified time.", result);
     }
+
+    [Fact]
+    public void ValidateUserDeleteBookingRequest_ValidBooking_ReturnsEmptyList()
+    {
+        // Arrange
+        var booking = GetBooking();
+        var userClaims = new UserClaims("name", booking.UserId, "noRole");
+
+        // Act
+        var result = BookingService.ValidateUserDeleteBookingRequest(booking, userClaims);
+
+        // Assert
+        Assert.Empty(result);
+    }
+
+    [Fact]
+    public void ValidateUserDeleteBookingRequest_UserNotAuthorized_ReturnsErrorMessage()
+    {
+        // Arrange
+        var booking = GetBooking();
+        var userClaims = new UserClaims("name", "someOtherUserId", "noRole");
+
+        // Act
+        var result = BookingService.ValidateUserDeleteBookingRequest(booking, userClaims);
+
+        // Assert
+        Assert.Single(result);
+        Assert.Contains("User is not authorized to delete the booking.", result[0]);
+    }
+
+    [Fact]
+    public void ValidateUserDeleteBookingRequest_UserIsEventAdmin_ReturnsEmptyList()
+    {
+        // Arrange
+        var booking = GetBooking();
+        var userClaims = new UserClaims("name", "userId123", "EventAdmin");
+
+        // Act
+        var result = BookingService.ValidateUserDeleteBookingRequest(booking, userClaims);
+
+        // Assert
+        Assert.Empty(result);
+    }
```

Comment:
hvor er Test case for "Booking with {booking.Id} not found"?

Reply:
Godt sett : )
La den til nå!